README.md

```diff
@@ -1,10 +1,14 @@
 CDK REST API template
 ---
 
-An example of project that uses Chemistry Development Kit (CDK) and provides a REST API. To start it in IDE, run [Starter](./src/main/java/cdkapi/Starter.java) and open your browser at http://localhost:8080/api/mw/CCC
+An example of project that uses Chemistry Development Kit (CDK) and provides a REST API.
+
+To start it in IDE, run [Starter](./src/main/java/cdkapi/Starter.java) and open your browser at http://localhost:8080/api/mw/CCC
+
+To compile and start from the command line, run `mvn clean install` followed by `java -jar target/cdk-rest-template-1.0-SNAPSHOT-shaded.jar`.
```

Comment:
`mvn clean package` creates the jar file in the target dir. `install` puts it into the local maven repo, it's not necessary most of the time. Updated it with the next commit.

Reply:
I never full understood Maven. Without what I added to the `pom.xml`, the `mvn package` did not create a jar with the dependencies.

BTW, it's working great! Thank you so much for this template repo!

Reply:
Yeah, you added the shade plugin - that one creates the jar-with-dependencies. It unpacks all jar files, and then repacks them into a single uber jar. I use the same procedure for my projects, but at some point I'd like to find a way to package jar files inside of the uber jar file (instead of repackaging them).

PS: I also added instructions to build a Docker image a few minutes ago.